src/WeedStore.js

```diff
@@ -0,0 +1,19 @@
+import { observable, action, computed, decorate } from "mobx"
+
+class WeedStore {
+    allStrains = [];
+    currentStrains = [];
+    
+}
+
+decorate(
```

Comment:
still not 100% clear on this but hoping it makes more sense as we progress through implementing mobX